source/WindowScroller/WindowScroller.example.js

```diff
@@ -9,19 +9,27 @@ import AutoSizer from '../AutoSizer'
 import shallowCompare from 'react-addons-shallow-compare'
 import styles from './WindowScroller.example.css'
 
-export default class AutoSizerExample extends Component {
+export default class WindowScrollerExample extends Component {
   static contextTypes = {
-    list: PropTypes.instanceOf(Immutable.List).isRequired
+    list: PropTypes.instanceOf(Immutable.List).isRequired,
+    customElement: PropTypes.object,
```

Comment:
`any` type is probably okay here. It's not great, but even [the official Flow types use `any` for `ref` type](https://github.com/facebook/flow/blob/7aa23dd3899e02c8d3cd7691890b268db913bd6b/lib/react.js#L125).